client/landing/splash.tsx

```diff
@@ -125,10 +125,21 @@ const ButtonsContainer = styled.div`
   flex-direction: row;
   flex-wrap: wrap;
   justify-content: center;
+  width: 100%;
+`
+
+const ContainerButtonAligner = styled.div.attrs(p => p)<{ align: 'left' | 'right' | 'center' }>`
+  flex: 1 1 0px;
+  display: flex;
+  justify-content: ${p => ({ center: 'center', left: 'flex-start', right: 'flex-end' })[p.align]};
+  @media (max-width: 800px) {
```

Comment:
What's the difference between this and `@media screen and (max-width: 800px)` that we use in the rest of the file?

Also, please add a blank space before this line.

Reply:
1. It was related to the proposed solution specifically — nvm
2. Atm how buttons wrap is controlled by their flex container so they may get a bit too stacked horizontally at some point ([like that](https://github.com/ShieldBattery/ShieldBattery/assets/102466089/ac58d7fa-c338-4807-a417-8468ba97ac2a)) — not really a problem